src/poetry/utils/env.py

```diff
@@ -1087,7 +1088,20 @@ def build_venv(
 
         args.append(str(path))
 
-        return virtualenv.cli_run(args)
+        cli_result = virtualenv.cli_run(args)
+
+        # Exclude the venv folder from from macOS Time Machine backups
+        # TODO: Add backup-ignore markers for other platforms too
+        if sys.platform == "darwin":
+            import xattr  # type: ignore[import]
```

Comment:
Instead of adding a `# type: ignore` comment let's add this to the Mypy settings in pyproject.toml

Reply:
Cool, added it there instead :)